pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1022,7 +1022,9 @@ func (f *frameworkImpl) runPreBindPlugin(ctx context.Context, pl framework.PreBi
 
 // RunBindPlugins runs the set of configured bind plugins until one returns a non `Skip` status.
 func (f *frameworkImpl) RunBindPlugins(ctx context.Context, state *framework.CycleState, pod *v1.Pod, nodeName string) (status *framework.Status) {
-	startTime := time.Now()
+	// On Windows, time.Now() is not as precise, 2 consecutive calls may return the same timestamp,
+	// resulting in 0 time delta / latency.
+	startTime := time.Now().Add(-time.Microsecond)
 	defer func() {
 		metrics.FrameworkExtensionPointDuration.WithLabelValues(bind, status.Code().String(), f.profileName).Observe(metrics.SinceInSeconds(startTime))
```

Comment:
is zero duration ever desirable in `SinceInSeconds`? Can we just implement a generic logic there to avoid the need to patch every place it may become an issue on Windows?

Reply:
Done